### DISCUSSION TITLE
Gaomon S620 Not Detected, Taking Forever to Find/Connect

### DISCUSSION THREAD

**Discussion by Zoruso** [OpenTabletDriver/OpenTabletDriver#2168] [Q&A] [answered]
So This Afternoon I wanted to play osu with a friend so I opened OpenTabletDrivers and went to go grab a drink. Once I came back 5 minutes later it was still not finding my tablet, I unplugged my tablet reinstalled the drivers etc and nothing seemed to work. So I took a nap, When I awoke then opened the tablet drivers I and waited 15 minutes to see if my computer was just slow but to no avail. Please fix this issue as its very annoying, I check debug and all I get is this plus long strings of text 
![image](https://user-images.githubusercontent.com/103226169/162338008-77df6ced-26d3-470e-a359-a3489faa735d.png)

**Comment by jamesbt365** [comment, marked as answer]
If you are experiencing crashing with OpenTabletDriver do this:
- First make sure that there is no Configurations folder if you are using any version higher than 0.5.3.3.
- Delete or move %localappdata%\OpenTabletDriver\settings.json
- Delete or move %localappdata%\OpenTabletDriver\Plugins
